cptv-convert.py

```diff
@@ -69,18 +70,19 @@ def convert_heat_to_img(frame, colormap, temp_min = 2800, temp_max = 4200):
     return img
 
 def main():
+    dir = os.path.dirname(os.path.realpath(__file__))
     parser = argparse.ArgumentParser()
 
-    parser.add_argument('-o', '--output-folder', help="Where to save mp4 files.")
-    parser.add_argument('-s', '--source-folder', help="Folder with cptv files to convert.")
-    parser.add_argument('-c', '--colormap', help="Colormap to use for conversion.")
+    parser.add_argument('source_folder', help="Folder with cptv files to convert.")
+    parser.add_argument('-o', '--output-folder', help="Where to save mp4 files.", default=join(dir, 'videos'))
```

Comment:
I was thinking the default behaviour would be to use a directory that's a subdirectory of `source_folder` not the directory where the code lives.

You'll have to do this as some extra logic after the call to `parse_args`.